recipes-containers/kubelet/files/launch-kubelet.sh

```diff
@@ -18,7 +18,7 @@
 # ----------------------------------------------------------------------------
 
 DEVICE_ID=`jq -r .deviceID /userdata/edge_gw_config/identity.json`
-if [ $? -ne 0 ]; then
+if [[ $? -ne 0 ]] || [[ $DEVICE_ID == null ]]; then
```

Comment:
Does "null" need to be encapsulated?

Reply:
I have tested this patch, it works without the quotes.